docs/pages/integrations/github/github.mdx

```diff
@@ -0,0 +1,61 @@
+import { Callout } from "nextra/components";
+
+# GitHub
+
+<Callout type="info">
+	Your email address has to be verified to create an access token.
+</Callout>
+
+1. Go to [GitHub](https://github.com) and log in.
+
+2. Click on your profile photo in the upper-right corner, then click **Settings**.
+
+3. In the left sidebar, click on **Developer settings**.
+
+4. In the left sidebar, under **Personal access tokens** select the kind of access token you want. Click [Here](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens#about-personal-access-tokens) for more information.
+
+<Callout type="info">
+	To access enterprise features (e.g. Audit logs), create an **Token
+	(classic)**.
+</Callout>
+
+5. Select **Generate new token**, then click on **Generate new token**
+
+![GitHub API](/github_api.png)
+
+6. Give your token a descriptive name.
+
+7. Set an Expiration for the token.
+
+8. Select which Repositories should be accessible with this token and select the Permissions you would like to give this token. For more information about specific permission, click [Here](https://docs.github.com/en/rest/authentication/permissions-required-for-fine-grained-personal-access-tokens?apiVersion=2022-11-28).
+
+<Callout type="info">
+	To access the Audit logs, you need at least the **audit_log** permission.
+</Callout>
+
+9. Click **Generate token**.
+
+10. Copy your token.
+
+11. Go to the **Settings** page in **Admyral**. Click on **Create New Secret** in the **Secrets** section.
+
+12. Give your credential a name, e.g. `GitHub`.
+
+13. The following secrets structure is expected:
+
+| Key            | Value                            |
+| -------------- | -------------------------------- |
+| `access_token` | Paste your **Access Token** here |
+
+14. Click on the **Save** button.
+
+Alternatively, you can use the following CLI command structure:
+
+<Callout type="info">
+	To access enterprise features (e.g. Audit logs), use
+	**github_enterprise_secret** instead
+</Callout>
```

Comment:
Confusing because `github_enterprise_secret` is just the name which can be arbitrary.